saleor/payment/gateways/razorpay/tests/test_razorpay.py

```diff
@@ -27,6 +27,7 @@ def gateway_config():
     return GatewayConfig(
         gateway_name="razorpay",
         auto_capture=False,
+        supported_currencies="USD",
```

Comment:
Do we check whether the provider supports to passed currency? `INR` is the only supported currency for RazorPay

Reply:
We decided to make supported currencies only configurable on the dashboard right now. There are plans to fetch supported currencies from gateway.

Reply:
Is there an easy way to use INR in this test case?

Reply:
I will replace here `supported_currencies` with `INR`.